apps/src/sites/studio/pages/home/teacher_homepage.js

```diff
@@ -9,16 +9,10 @@ function showTeacherHomepage() {
   const coursesData = document.querySelector('script[data-courses]');
   const configCourses = JSON.parse(coursesData.dataset.courses);
 
-  const courses = configCourses.map(course => (
-      {
-        key: course.id,
-        courseName: course.name,
-        description: course.description,
-        image: course.image,
-        link: course.link,
-        assignedSections: []
-      }
-  ));
+  const sectionsData = document.querySelector('script[data-sections]');
+  const configSections = JSON.parse(sectionsData.dataset.sections);
+  console.log(configSections);
```

Comment:
Shouldn't leave logging in.. some browsers fail on this if dev console not open.